### ISSUE TITLE
Sponsor app not working

### ISSUE THREAD

**Issue by Roosted7** [SHA2017-badge/micropython-esp32#78]
Anyone already on this?

**Comment by annejan** [comment]
Yes, this will be fixed after #75 since that enables us to load PNGs from the same FS as the micropython stuff . .

**Comment by annejan** [comment]
Created new issue for shipping sponsors app and more https://github.com/SHA2017-badge/auto-flasher/issues/5

**Comment by Roosted7** [comment]
Fixed :+1:
